content/200-concepts/100-components/02-prisma-client/090-raw-database-access.mdx

```diff
@@ -17,6 +17,8 @@ Prisma Client exposes two methods that allow you to send raw SQL queries to your
 
 - Use `$queryRaw` to return actual records (for example, using `SELECT`)
 - Use `$executeRaw` to return a count of affected rows (for example, after an `UPDATE` or `DELETE`)
+- Use `$queryRawUnsafe` to return actual records (for example, using `SELECT`) using a raw string. **Potential SQL injection risk**
+- Use `$executeRawUnsafe` to return a count of affected rows (for example, after an `UPDATE` or `DELETE`) using a raw string. **Potential SQL injection risk**
```

Comment:
Adding these two here, makes the "two methods" mention above invalid. (Can not comment on line 16 though in GH)